compiler-rt/cmake/config-ix.cmake

```diff
@@ -223,7 +223,7 @@ set(COMPILER_RT_SUPPORTED_ARCH)
 # runtime libraries supported by our current compilers cross-compiling
 # abilities.
 set(SIMPLE_SOURCE ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/simple.cc)
-file(WRITE ${SIMPLE_SOURCE} "#include <stdlib.h>\n#include <stdio.h>\nint main(void) { printf(\"hello, world\"); }\n")
+file(WRITE ${SIMPLE_SOURCE} "#include <stdint.h>\nint main(void) { return 0; }\n")
```

Comment:
Interesting, I have a cache with the failure but after nuking the directory I don't get it anymore. It might've been stuck from a previous run and didn't get cleared properly after updating something else. Sorry about that.

After some other hacks I think I've gotten it to at least pass configuring, building only creates files for a few of the floating point values. This seems to be far off the "generic sources" list I see.
```
extendbfsf2.c.o
truncdfbf2.c.o
truncsfbf2.c.o
```

I can go ahead and close this since it doesn't seem to be necessary for the parts I'm interested in.